kde-apps/spectacle/spectacle-24.02.49.9999.ebuild

```diff
@@ -53,10 +53,7 @@ DEPEND="${COMMON_DEPEND}
 	>=dev-libs/plasma-wayland-protocols-1.11.1
 "
 RDEPEND="${COMMON_DEPEND}
-	|| (
-		>=dev-qt/qttools-${QTMIN}:6[qdbus]
-		dev-qt/qdbus:0
-	)
+	>=dev-qt/qttools-${QTMIN}:6[qdbus]
```

Comment:
That does not pull in Qt5 as far as I can see.

Reply:
I was looking for everything using qdbus, spectical doesn't appear to use any of the libraries in qttools either or list it in https://invent.kde.org/graphics/spectacle/-/blob/master/CMakeLists.txt?ref_type=heads

Reply:
I'll see about removing the other line too

Reply:
Same as with kinfocenter, this is about the qdbus binary used at runtime, hence being a runtime dependency only.